src/lib/components/ResultsTable/ResultsTable.js

```diff
@@ -137,4 +147,5 @@ ResultsTable.defaultProps = {
   totalHitsCount: 0,
   showMaxRows: invenioConfig.APP.DEFAULT_RESULTS_SIZE,
   renderEmptyResultsElement: null,
+  hideFooterMessage: false,
```

Comment:
Is this prop used anywhere? I don't see it used in this PR.
For readability, I would also avoid the double negation in `if` conditions, `hide` and `false`. Why not:
```suggestion
  showFooterMessage: true,
```

Reply:
The property is used in cds-ils (example in this PR: https://github.com/CERNDocumentServer/cds-ils/pull/656). I know we don't want to modify invenio for changes needed by cds, but the other potential solutions are quite bad; plus I think it makes a lot of sense to hide a footer that becomes redundant an even confusing once you add pagination to the table.